broker-core/src/main/java/io/zeebe/broker/transport/TransportComponent.java

```diff
@@ -69,7 +67,6 @@ private void createClientTransports(final SystemContext context) {
     final NetworkCfg networkCfg = brokerCfg.getNetwork();
     final int nodeId = brokerCfg.getCluster().getNodeId();
     final SocketAddress managementEndpoint = networkCfg.getManagement().toSocketAddress();
-    final SocketAddress subscriptionEndpoint = networkCfg.getSubscription().toSocketAddress();
```

Comment:
We can also remove subscription from the configuration. It is not needed anymore?

Reply:
Yeah agree can do. I thought we will do this in https://github.com/zeebe-io/zeebe/issues/1928